docs/contributor/03-11-hap-rules.md

```diff
@@ -4,11 +4,11 @@
 > This feature is still being developed and will be available soon.
 
 The Hyperscaler Account Pool (HAP) rule configuration is a functionality that allows you to control pool selection from a configurable helm property. 
-A pool is a set of Gardener's SecretBinding resources that share the same search labels. 
-Pool selection is a process that decides which SecretBinding pool is the most suitable for every SAP BTP, Kyma runtime creation request.
-The process generates a list of [search labels](#search-labels) based on Kyma runtime's attributes.
+A pool is a set of Gardener's SecretBinding custom resources that share the same labels. 
+Pool selection is a process that decides which Secret Binding pool is the most suitable for every SAP BTP, Kyma runtime creation request.
```

Comment:
I'm not sure about this change. For service instances (and service instances), we agreed to follow the Help Portal guideline and use natural language without capitalization. We may refer to the resource using camel case, but then it's ServiceInstance or ServiceInstance resource(s). So following this pattern, I'd use `Secret binding` or `SecretBinding`. WDYT?

Reply:
It's a K8s custom resource, its kind would be **SecretBinding**, so let's keep that for consistency.